media/css/firefox/independent-splash.less

```diff
@@ -4,15 +4,6 @@
 
 @import "../sandstone/lib.less";
 
-@font-face {
```

Comment:
Looks like `independent-splash.less` is always with `mozilla-share-cta.less` but I'll keep the `@font-face` just in case.